src/OrbitGl/App.h

```diff
@@ -452,6 +452,9 @@ class OrbitApp final : public DataViewFactory, public orbit_capture_client::Capt
 
   [[nodiscard]] bool HasFrameTrackInCaptureData(uint64_t instrumented_function_id) const;
 
+  enum class SelectionMode { kFirst, kLast, kMin, kMax };
+  void SelectTextBoxAndZoom(uint64_t function_id, SelectionMode selection_mode);
```

Comment:
I'm undecided if this function should be inside TimeGraph or CaptureWindow, but I think that it shouldn't be inside App.

Reply:
All the other functions of this type are currently in App (See SelectTextBox for example) and it needs to be in App if we want to introduce `AppInterface` as discussed in the doc.

It might move somewhere else in the long run but for now there is no way around it.

Reply:
Ok. I will include some wording in the last commit about why we are moving this method to App, and our next steps.

Reply:
Done. PTAL